app/archive.js

```diff
@@ -17,6 +17,9 @@ function isDupe(newFile, array) {
 export default class Archive {
   constructor(files = []) {
     this.files = Array.from(files);
+    this.timeLimit = DEFAULTS.EXPIRE_SECONDS;
+    this.dlimit = 1;
+    this.password = null;
```

Comment:
I moved these fields in from choo `state`. It made the metric call easier with the data in one place, and it seems like a better place anyway